client/src/styles/bundle.scss

```diff
@@ -2,10 +2,16 @@
 @import "variables";
 
 // Thirdparty
+@import "~bootstrap/scss/functions";
 @import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/variables";
 
 // Elemental components
 @import "admin";
 @import "history";
 @import "reports";
 @import "textcheckboxgroup";
+@import "../components/ElementEditor/Content";
+@import "../components/ElementEditor/ElementEditor";
+@import "../components/ElementEditor/Header";
```

Comment:
Awesome! I see you've added the Header.scss file for example - is there a corresponding amount of dead SCSS in the original stylesheet that can be deleted now?